ui/helpers/utils/util.js

```diff
@@ -705,3 +705,16 @@ export const fetchTokenExchangeRates = async (
     return {};
   }
 };
+
+export const hexToText = (hex) => {
+  if (!hex) {
+    return hex;
+  }
+  try {
+    const stripped = stripHexPrefix(hex);
```

Comment:
Can we use `@metamask/utils` instead of defining new functions?

Reply:
I could not find an equivalent method in `@matamask/utils`.

Reply:
I would recommend `hexToBytes` and `bytesToString`, though I'm not sure I fully understand the conditionals in this utility function,

Reply:
@jpuri thoughts on the above?
Also for consistent behaviour we should replace the `msgHexToText` in the [`SignatureRequestOriginal` component](https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/confirmations/components/signature-request-original/signature-request-original.component.js#L106)

Reply:
@cryptotavares : I would avoid changing original confirmation components. As we work on more confirmations, we will come across many situations like this to keep code consistent, but it will be hard to change + test existing stuff also as we make new. Ans, also eventually we will g et rid of old components.

Reply:
We have agreed that if it is a straightforward duplication then yes we should update current confirmations components with the shared code (instead of keeping the duplication)... it should simplify maintenance (for the time where we have to support old and new components). But the moment the logic is no longer a duplication and branches off somewhere, then we shouldn't update the old components (as we do not want to couple any new code paths that we are adding to old components, making it harder to get rid of them once the time comes).